### PR TITLE
ACM-2781: Have operator as a prerequisite

### PR DESCRIPTION
**Pull request by chenz4027**
Signed-off-by: Maggie Chen <magchen@redhat.com>
https://issues.redhat.com/browse/ACM-2781

### CODE REVIEW

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

**Comment by chenz4027** [comment]
Tagging QE, doc and PX
QE Approver:
 /assign @yanpzhan
Docs Approver:
 /assign @opayne1
PX Approver:
/assign @RickJWagner

File: quickstarts/host-inventory-quickstart.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ spec:
   durationMinutes: 20
   description: Configure your host inventory and create a cluster.
   prerequisites: 
+    - 'You completed the "Get started with multicluster engine"'
```
Comment (opayne1) [inline quickstarts/host-inventory-quickstart.yaml:15]: Just a suggestion for clarity:
```suggestion
    - 'You have completed the steps to "Get started with multicluster engine"'
```

**Review by opayne1 — COMMENTED** [review]
Just a small suggestion for clarity.

**Comment by yanpzhan** [comment]
Checked on cluster launched against the pr, there is 'You have completed the steps to "Get started with multicluster engine" quick start.' in the prerequisites for host inventory quick start.
/label qe-approved

**Review by jhadvig — APPROVED** [review]
/lgtm

**Pull request merged**